### PR TITLE
Module names are now clickable in sidebar and link to module view.

### PR DESCRIPTION
**Pull request by kennyczadzeck**
Module names are now clickable in sidebar and link to module view.

### CODE REVIEW

**Comment by tesk9** [comment]
Code looks good.
I don't know if this is relevant to this pull request or not, but I checked the links from /#/library/1, and from there I got sent back to /#/admin/library/1. This strikes me as not the behavior that we want, right? Is that something that needs to get fixed by whoever is working on the view-as button?
That being said, the links do link to the modules they're intended to link to, even if it does knock me into the admin view automatically.

**Comment by kennyczadzeck** [comment]
The links in the library view are a separate pull request issue. They are not related to the sidebar as it's a directive.

File: assets/js/templates/directives/admin_sidebar_template.html [hunk 1]
```diff
@@ -1,6 +1,6 @@
 <div class="panel">
   <div ng-repeat="module in modules">
-    <h4><strong>{{module.name}}</strong></h4>
+    <h4><strong><a href="/#/admin/library/{{module.library.id}}/module/{{module.id}}">{{module.name}}</a></strong></h4>
```
Comment (tesk9) [inline assets/js/templates/directives/admin_sidebar_template.html]: use ng href

File: assets/js/templates/directives/student_sidebar_template.html [hunk 2]
```diff
@@ -1,6 +1,6 @@
 <div class="panel">
   <div ng-repeat="module in modules">
-    <h4><strong>{{module.name}}</strong></h4>
```
Comment (tesk9) [inline assets/js/templates/directives/student_sidebar_template.html]: ng-href

**Comment by kennyczadzeck** [comment]
added ng-href to all interpolated hyperlink paths.

**Pull request merged**